internal/pkg/gateway/api.go

```diff
@@ -175,16 +175,25 @@ func (gs *Server) Submit(ctx context.Context, request *gp.SubmitRequest) (*gp.Su
 	}
 
 	orderer := orderers[0] // send to first orderer for now
+
+	broadcast, err := orderer.client.Broadcast(ctx)
+	if err != nil {
+		return nil, rpcError(
+			codes.Aborted,
+			"failed to send transaction to orderer",
```

Comment:
This error is the same as the next one and that could complicate debug. It’s also slightly inaccurate as this is a failure to create an `AtomicBroadcast_BroadcastClient` or “BroadcastClient”.